Student/StudentE2ETests/Discussions/DiscussionsTest.swift

```diff
@@ -22,6 +22,12 @@ class DiscussionsTests: E2ETestCase {
     typealias Helper = DiscussionsHelper
     typealias DetailsHelper = Helper.Details
     typealias ReplyHelper = DetailsHelper.Reply
+    typealias NewDiscussion = Helper.NewDetails
+
+    override func tearDown() {
+        let featureFlagResponse = seeder.setFeatureFlag(featureFlag: .newDiscussion, state: .off)
+        XCTAssertEqual(featureFlagResponse.state, DSFeatureFlagState.off.rawValue)
```

Comment:
`super` isn't called after we tore down our state. Was this on purpose?

Reply:
Nice catch, thanks! 👍